lib/src/base-styles/_nx-lists.scss

```diff
@@ -171,59 +167,84 @@
 /*
  * Action or Clickable Lists
  */
-.nx-list--clickable {
-  .nx-list__item {
 
-    &, > .nx-list__link {
-      &.selected {
-        &, &:hover {
-          background-color: var(--nx-color-interactive-background-selected);
-        }
-      }
+.nx-list__item--link, .nx-list__item--clickable {
+  display: contents;
+}
 
-      &:hover {
-        background-color: var(--nx-color-interactive-background-hover);
-        cursor: pointer;
+.nx-list__item--clickable, // preferred
+.nx-list__item--link,
```

Comment:
This one is deprecated too right?

Reply:
> This one is deprecated too right?

Yeah, in my mind this was sort of a soft deprecation, but noted here: https://github.com/sonatype/sonatype-react-shared-components/pull/226/commits/0df257c287abae5db9943f4590e9739269cac2a0